### PR TITLE
#DP-49 ft(orgAuth): Organization login pages

### PR DESCRIPTION
**Pull request by Julesbi**
#### What does this PR do?

- [X] Page for entering an organization's URL is implemented

- [X] The sign in page is implemented

#### How should this be manually tested?

- git clone https://github.com/atlp-rwanda/atlp-pulse-fn.git
- cd atlp-pulse-fn
- git checkout ft-Org-login-page-dp-49
- npm install
- npm run dev

#### What are the relevant pivotal tracker stories?
[DP-49](https://trello.com/c/PyHzBwSl/49-org-login-page)

#### Screenshots (if appropriate)

Screenshot (737)

### CODE REVIEW

File: src/pages/Organization/Orggooglelogin.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            Or
+          </div>
+
+          <button className="w-full py-3 px-4 bg-primary hover:bg-cyan-700 rounded-md text-white font-bold sm:py-1-px-1 sm:font-thin md:text-[14px]">
+            Sign in with email and password
+          </button>
+
+          <div className="w-full text-sm  text-light-text dark:text-dark-text-fill sm:font-thin">
+            Looking to register an organization instead?
+            <a className="text-primary">Register a new organization</a>
```
Comment (samyshyaka) [inline src/pages/Organization/Orggooglelogin.tsx:37]: Remember to link this to the Organization registration page

**Review by ceelogre — COMMENTED** [review]
Have the team first approve the PR before adding me as a reviewer.

**Review by jeandamascyifuzo — APPROVED** [review]
Good job @Julesbi

File: src/pages/Organization/Orggooglelogin.tsx (hunk 1 above)
Comment (ceelogre) [reply src/pages/Organization/Orggooglelogin.tsx:37]: Has this been done? I don't see how to link to this page.

**Review by ceelogre — COMMENTED** [review]
There's a comment by Sam and I. Let us know how it's been addressed.

File: src/pages/Organization/Orggooglelogin.tsx (hunk 1 above)
Comment (Julesbi) [reply src/pages/Organization/Orggooglelogin.tsx]: It is now solved

**Review by feyton — APPROVED** [review]
## Approval

**Pull request merged**